test/services/link/index.test.js

```diff
@@ -1,10 +1,16 @@
+/* eslint-disable func-names */
 const assert = require('assert');
 const chai = require('chai');
 const chaiHttp = require('chai-http');
 const app = require('../../../src/app');
 
 chai.use(chaiHttp);
 
+const subtractDays = (date, d) => {
+	date.setTime(date.getTime() - (d * 24 * 60 * 60 * 1000));
```

Comment:
this day subtraction method differs to https://github.com/schul-cloud/schulcloud-server/pull/680/files#diff-0d40d91045f298d8a0c6c8a49036c54eR25 one option should be enough

Reply:
as spoken about, function will be left different intentionally to check for possible conflicts